generic/tests/netperf.py

```diff
@@ -148,8 +150,19 @@ def _pin_vm_threads(vm, node):
     vm.verify_alive()
     login_timeout = int(params.get("login_timeout", 360))
 
-    session = vm.wait_for_login(timeout=login_timeout)
+    server_ip = vm.wait_for_get_address(0, timeout=5)
+    if len(params.get("nics", "").split()) > 1:
+        vm.wait_for_serial_login(timeout=10, restart_network=True)
+        server_ctl = vm.wait_for_login(nic_index=1, timeout=login_timeout)
+        server_ctl_ip = vm.wait_for_get_address(1, timeout=5)
+        session = vm.wait_for_login(nic_index=1, timeout=30)
+    else:
+        server_ctl = vm.wait_for_login(timeout=login_timeout)
+        server_ctl_ip = server_ip
```

Comment:
remove line 153, and move it into line 161 just like line 158 did.

Reply:
line 153 is "server_ip" which means it will be used in netperf -H "ip", it's same unique in both mulit-nics or singe nic.